ee/clickhouse/models/test/test_filters.py

```diff
@@ -62,32 +62,56 @@ def test_simplify_cohorts(self):
         filter = Filter(data={"properties": [{"type": "cohort", "key": "id", "value": cohort.pk}]})
 
         self.assertEqual(
-            filter.simplify(self.team).properties_to_dict(),
-            {"properties": [{"key": "email", "operator": "icontains", "value": ".com", "type": "person"},]},
+            filter.simplify(self.team).properties_to_dict(), {},
```

Comment:
think I'll carve simplify changes out to a separate PR. We want these tests for both old and new style properties, for the time being. And until migration completes, simplify using old properties is perfectly fine.